distributions/Dist_saw533.py

```diff
@@ -0,0 +1,47 @@
+
+import numpy as np
+from base_distribution import BaseDistribution
+
+class Dist_saw533(BaseDistribution):
+	def __init__(self):
+		self.f_max = 1
+		self.x_min = -1
+		self.x_max = 1
+
+
+	def pdf(self, x):
+		"""This is your PDF"""
+		return (0.5*x)+0.5
+
+	def mean(self):
+		"""This is the mean of the PDF"""
+		return 0.33333333333333
+
+	def std(self):
+		"""This is the standard deviation of the pdf"""
+		return np.sqrt(0.22222222222)
```

Comment:
similarly, should avoid 0.2222222 and use 2./9

Note 2/9 will use integer division and give you 0. You need the . after the 2